USAGE.md

```diff
@@ -85,48 +85,29 @@ The verification process validates the agent signature against the identity prov
 These operations are implemented using [Sigstore](https://www.sigstore.dev/).
 
 ```bash
-## Sign the agent data model
-cat model.json | dirctl sign --stdin > signed.model.json
-
-## Verify agent data models
-cat model.json | dirctl verify --stdin
-cat signed.model.json | dirctl verify --stdin
-
-## Verify signature using custom parameters:
-# 1. Only trust users with "cisco.com" addresses
-# 2. Only trust issuers from "github.com"
-dirctl verify signed.model.json \
-   --oidc-identity "(.*)@cisco.com" \
-   --oidc-issuer "(.*)github.com(.*)"
-
-## Replace the base agent model with a signed one
-rm -rf model.json
-mv signed.model.json model.json
+## Push record with signature
+dirctl push record.json --sign
+
+## Verify record
+dirctl verify <record-cid>
```

Comment:
are we not capable of verifying anymore who the user is or what the signing method was? IMO this prevents us from answering the questions like: "is this record coming from this user with this key/identity" or "was this record SAFELY signed (like when using keyless OIDC signing)"